controllers/networking-calico/example/20-cluster.yaml

```
@@ -0,0 +1,27 @@
+---
```

Can you merge this content into `20-network.yaml` and delete this file? Makes it easier to use.

Also, please add: 

```suggestion
---
apiVersion: v1
kind: Namespace
metadata:
  name: shoot--foobar--aws
  labels:
    shoot.gardener.cloud/provider: azure
    networking.shoot.gardener.cloud/provider: calico
---
```